client/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "Name-Request",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/bcgov/namerequest.git"
+  },
+  "version": "0.1.0",
+  "private": true,
+  "license": "Apache License, Version 2.0",
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "test:unit": "vue-cli-service test:unit",
+    "test:e2e": "vue-cli-service test:e2e",
+    "lint": "vue-cli-service lint"
```

Comment:
You might want to add a script entry to run lint without changing your files automatically:
`"lint:nofix": "vue-cli-service lint --no-fix",`